Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/Customizations/AuthTokenGenerator.swift

```diff
@@ -16,7 +16,7 @@ import SmithyIdentity
 import struct Foundation.Date
```

Comment:
This file is the underlying auth token generator in runtime library, used by RDS and DSQL service auth token generators.